app/package.json

```diff
@@ -26,7 +26,6 @@
     "react-native-gesture-handler": "^2.13.2",
     "react-native-pager-view": "^5.4.25",
     "react-native-reanimated": "link:../",
```

Comment:
Why are we doong this? Removing this from app may cause vscode to consider all imports of "react-native-safe-area-context" inside app to be bugs.